bsip-00XX Sharedrop Operation.md

```diff
@@ -0,0 +1,153 @@
+
+## Sharedrop vesting stake 
+Sharedrop vesting stake will be a flat fee determined by the committee. It will be charged in BTS.
+
+It will be in BTS only and it will be held on a vesting balacne under a specific sharedrop vesting policy  for the period of **sharedrop_vesting_period (in weeks)** defined by the committee. At the end of the sharedrop_vesting_period, a sharedrop originator will be able to retrieve his funds from the vesting account. 
+
+Let's say a vesting period is 2 months.
+If a sharedrop originator does another sharedrop before the vesting period is over, the **sharedrop_vesting_period** counter resets and these 2 months will start from fresh.
+
+### Complicated vesting cases, when the committee updates sharedrop vesting parameters.
+- Vesting stake increases. In this case **dry_run_sharedrop** operation warns the user that he should have more money prepared. And then the sharedrop operation will put more money into vesting. 
+
+- Vesting stake decreases. Then the extra money becomes instantly available for withdrawal.  Balance availability is checked during vesting withdrawal operation.
+
+-  **Sharedrop_vesting_period** decreases or increases. The new vesting period is updated during the next sharedrop operation. 
+
+## Technical specifications
+```
+struct sharedrop_result {
+	// cost of sharedrop
+	share_type fee;
+	// required amount on the vesting balance
+	share_type vesting_stake;
+	// money may be claimed after this time
+	fc::time_point_sec vesting_period;
+}
+```
+
+
+There is a function that helps to fill parameters for sharedrop operation
+
+```
+// function that helps to fill parameters for sharedrop operation
+// returns  sharedrop_result
+sharedrop_result dry_run_sharedrop(asset sharedrop_asset, asset_id_type stake_asset,
+share_type min_stake_asset_amount) 
+```
+
+For sharedrop vesting balance add specific type for vesting_balance_object
+
+```
+struct sharedrop_vesting_policy
+{
+	// money may be claimed after this time  
+	fc::time_point_sec  start_claim;
+}
+```
+
+After each sharedrop operation start_clam will be updates as start_claim =TODAY + sharedrop_vesting_period
+
+
+# Discussion
+1. Flag **allow_sharedrop** for an asset. Should we have it?
+Each asset should have **allow_sharedrop** flag (TRUE/FALSE) that would either allow sharedrop to the holders of this asset or not allow it.
+
+2. One of the ideas is instead of making flat fee vesting for all sharedrop transactions, for each sharedrop transaction we put into vesting an amount based on the stakeholder number. And we have to put a new amount for every sharedrop transaction.
+In this case, it might quickly become too expensive for some users.
+
+
+# See Also
+
+https://github.com/bitshares/bsips/blob/master/bsip-0020.md
+
+## Copyright
```

Comment:
Please add a easy-to-understand Summary for the Shareholders

What are the additional leverages created for BTS holders, what are the benefits? Also, where are the risks?

Reply:
Addressing why BSIP 20 was deferred is relevant too, I believe it's because the BSIP terminology was about dividends/profit-sharing of fees instead of sharedropping as a form of issuance? If the reason for deference was related to regulations then perhaps BTS and committee owned assets could be excluded?